robolectric/src/test/java/org/robolectric/shadows/ShadowBluetoothDeviceTest.java

```diff
@@ -287,6 +287,7 @@ public void createRfcommSocketToServiceRecord_returnsSocket() throws Exception {
 
   @Test
   @Config(maxSdk = VERSION_CODES.R)
+  @SuppressWarnings("RethrowReflectiveOperationExceptionAsLinkageError")
```

Comment:
Why suppressing instead of changing code here?

Reply:
oops, thanks for spotting that now removed it.